examples/upsert-resource-at-path/.expected/config.yaml

```diff
@@ -0,0 +1 @@
+runCount: 2
```

Comment:
Do we have to specify this ? it should be idempotent by default.. right.

Reply:
I don't think we need this file.